app/src/main/java/ch/epfl/skysync/components/ConfirmFlightDetailUI.kt

```diff
@@ -119,6 +125,41 @@ fun ListElementText(text: String, padding: Dp, testTag: String) {
   )
   Spacer(modifier = Modifier.padding(4.dp))
 }
+
+@Composable
+fun HyperLinkText(title: String, value: String, padding: Dp) {
+  val uriHandler = LocalUriHandler.current
+  val googleMapsLink = "https://www.google.com/maps/search/?api=1&query=${value.replace(" ", "+")}"
```

Comment:
Could be more careful to do:
val encodedValue = URLEncoder.encode(value, "UTF-8")
val googleMapsLink = "https://www.google.com/maps/search/?api=1&query=$encodedValue"
In case there are other special characters in the value string

Reply:
thank you